ui/components/app/home-notification/index.scss

```diff
@@ -49,6 +49,7 @@
   &__checkbox {
     height: 13px;
     width: 13px;
+    vertical-align: middle;
     font-size: 16px;
     cursor: pointer;
```

Comment:
We should be able to remove these styles altogether, same for the label if it has any the component should take care of it